_posts/2020-01-28-hacking-2020.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title: Berlin Hack & Tell \#77 - Hacking 2020
+date: 2020-01-28
+time: '19:15'
+location: '[c-base](https://www.c-base.org)'
+meetupUrl: https://www.meetup.com/de-DE/Berlin-Hack-and-Tell/events/268089955/
+---
+
+* [Vote2p](https://github.com/gimlet2/vote2p) (voting, ipfs, pubsub) by [[Andrey](https://github.com/gimlet2)
+* [FPGA Badge Hack](https://hackaday.io/page/6764-hackaday-supercon-badge-boots-linux-using-sdram-cartridge) - Linux on RISC-V by [Drew Fustini](https://github.com/pdp7)
+* [obfuscated compressor](https://ioccc.org/2019/diels-grabsch1/hint.html) by Volker [Diels-Grabsch](https://njh.eu)
+* [supericon](https://github.com/momonala/supersizeme) by [Mohit Nalavadi](https://github.com/momonala)
```

Comment:
```suggestion
* [supericon](https://github.com/momonala/supersizeme) by [Mohit Nalavadi](https://github.com/momonala)

* [Super Resolution HTTP API](https://github.com/momonala/supersizeme) by [Mohit Nalavadi](https://github.com/momonala)
```

Reply:
Just a name change. Thanks again for the event! Had a nice time.

Reply:
Applied - thanks - great to hear!